### ISSUE TITLE
In Scala + gradle, error highlight doesn't work

### ISSUE THREAD

**Issue by tribbloid** [redhat-developer/lsp4ij#720]
![Image](https://github.com/user-attachments/assets/c98658ba-f753-4dad-afea-9ce17f5abcbc)

there is no trace or log error, how did this happen?

**Comment by tribbloid** [comment]
this error was encountered at: https://github.com/tribbloid/scaffold-gradle-kts/tree/b421ae8cb778affaf3759a9b79b9462e161dd9ed

**Comment by angelozerr** [comment]
> ![Image](https://github.com/user-attachments/assets/c98658ba-f753-4dad-afea-9ce17f5abcbc)
> 
> there is no trace or log error, how did this happen?

Have you other lsp features like completion ?

I suggest that you set traces to verbose and check in the lsp console that yiu have lsp publishDiagnostic traces

**Comment by tribbloid** [comment]
found the reason:![Image](https://github.com/user-attachments/assets/b20b1b3b-d942-4d12-9004-3de58aa76160)

I have to change it back to "built-in"

**Comment by angelozerr** [comment]
You mean that it exists an IJ plugin which is based on LSP4IJ for scala?

**Comment by tribbloid** [comment]
oh sorry, the above screenshot is from the official Scala plugin from JetBrains

If error highlighting is set to "Compiler" it will ignore the report from metaLS

**Comment by angelozerr** [comment]
If I understand correctly the jetbrains scala setting can disable validation from LSP4IJ ?

**Comment by tribbloid** [comment]
yes that's my experience so far, I could speculate that most people needing LSP4IJ & metaLS integration also rely on IntelliJ Scala plugin for other features

**Comment by angelozerr** [comment]
Do you see some publichDiagnostic in the LSP console? If no it means that error is not managed by metal LS.

**Comment by tribbloid** [comment]
@angelozerr unfortunately this is not possible at the moment. metaLS had an issue that prevents the console from launching, see https://github.com/scalameta/metals/issues/7072 for details

are they also in traces or logs? can't see anything marked as Diagnostic

**Comment by angelozerr** [comment]
> @angelozerr unfortunately this is not possible at the moment. metaLS had an issue that prevents the console from launching, see https://github.com/scalameta/metals/issues/7072 for details
> 
> are they also in traces or logs? can't see anything marked as Diagnostic

If you dont see any lsp publishDiagnostic  traces it is normal that you have no validation error.

Perhaps metal ls expects some config to enable validation?

The error validation that you have comes from the other scala intellij plugin, there is is no links with lsp4ij
